_site/images/bla.txt

```diff
@@ -0,0 +1 @@
+To be deleted
```

Comment:
What's this for?

Reply:
I thought I'd deleted it, will try to get rid of it again!